### PR TITLE
[16.0][MIG] sale_manual_delivery: Migration to 16.0

### PR DESCRIPTION
**Pull request by nguyenhk99**
Based upon:
* #1983

### CODE REVIEW

**Review by StephaneMangin — APPROVED** [review]

**Review by leemannd — APPROVED** [review]

File: sale_manual_delivery/tests/test_manual_delivery.py [hunk 1]
```diff
@@ -289,7 +289,7 @@ def test_03_sale_selected_lines(self):
             'Picking should be created after "manual delivery" wizard call',
         )
         self.assertEqual(
-            len(order3.picking_ids.move_lines),
+            len(order3.picking_ids.move_line_ids),
```
Comment (grindtildeath) [inline sale_manual_delivery/tests/test_manual_delivery.py]: :warning: `move_lines` refers to `stock.move` whereas `move_line_ids` refers to `stock.move.line`.
I guess, here you should use `move_ids`: https://github.com/odoo/odoo/blob/16.0/addons/stock/models/stock_picking.py#L346

File: sale_manual_delivery/tests/test_manual_delivery.py [hunk 2]
```diff
@@ -412,7 +412,7 @@ def test_03_sale_multi_delivery(self):
             ' after "manual delivery" wizard call with same date',
         )
         self.assertEqual(
-            sum(first_picking.mapped("move_lines.product_uom_qty")),
+            sum(first_picking.mapped("move_line_ids.reserved_uom_qty")),
```
Comment (grindtildeath) [inline sale_manual_delivery/tests/test_manual_delivery.py]: Same as above, AFAIU we want to test the quantity of each move that is created, not how much was reserved.

Comment (nguyenhk99) [reply sale_manual_delivery/tests/test_manual_delivery.py]: Hello @grindtildeath , Thank for your review. The PR has been updated.

**Pull request merged**